### PR TITLE
New package: DWANGO.OpenToonz.Nightly version 2023.10.31

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Related https://github.com/microsoft/winget-pkgs/issues/124865
- Related https://github.com/microsoft/winget-pkgs/pull/125044

This is a nightly build version. It seems the stable version will cause unexpected crash. Submit this version to see if it will crash as stable version.

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/126821)

### CODE REVIEW

File: manifests/d/DWANGO/OpenToonz/Nightly/2023.10.31/DWANGO.OpenToonz.Nightly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 2023.10.31
+PackageLocale: en-US
+Publisher: DWANGO Co., Ltd.
+PublisherUrl: https://en.dwango.co.jp/
+PrivacyUrl: https://en.dwango.co.jp/terms/privacy/
+Author: opentoonz
+PackageName: OpenToonz
+PackageUrl: https://github.com/opentoonz/opentoonz_nightlies
+License: BSD 3-Clause "New" or "Revised" License
+LicenseUrl: https://github.com/opentoonz/opentoonz_nightlies/blob/master/LICENSE.txt
```
Comment (Dragon1573) [inline manifests/d/DWANGO/OpenToonz/Nightly/2023.10.31/DWANGO.OpenToonz.Nightly.locale.en-US.yaml:14]:
```suggestion
LicenseUrl: https://github.com/opentoonz/opentoonz/blob/master/LICENSE.txt
```

**Review by Dragon1573 — COMMENTED** [review]
The repository only hosts nightly build binaries. It does not contain any source code.

File: manifests/d/DWANGO/OpenToonz/Nightly/2023.10.31/DWANGO.OpenToonz.Nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Publisher: DWANGO Co., Ltd.
+PublisherUrl: https://en.dwango.co.jp/
+PrivacyUrl: https://en.dwango.co.jp/terms/privacy/
+Author: opentoonz
+PackageName: OpenToonz
+PackageUrl: https://github.com/opentoonz/opentoonz_nightlies
+License: BSD 3-Clause "New" or "Revised" License
+LicenseUrl: https://github.com/opentoonz/opentoonz/blob/master/LICENSE.txt
+Copyright: Copyright (c) 2016 - 2023, DWANGO Co., Ltd.
+CopyrightUrl: https://github.com/opentoonz/opentoonz_nightlies/blob/master/LICENSE.txt
```
Comment (Dragon1573) [inline manifests/d/DWANGO/OpenToonz/Nightly/2023.10.31/DWANGO.OpenToonz.Nightly.locale.en-US.yaml:16]:
```suggestion
CopyrightUrl: https://github.com/opentoonz/opentoonz/blob/master/LICENSE.txt
```

**Comment by Dragon1573** [comment]
@stephengillie - Could you please manually validate this package? I simply test this version in my Windows Sandbox and unable to make it crash as stable version 1.7.1.

I think this version is acceptable.

**Comment by stephengillie** [comment]
Unfortunately, the Nightly has the same error as the [other version](https://github.com/microsoft/winget-pkgs/pull/125044):

image

**Comment by Dragon1573** [comment]
According to [Dev Team's response](https://github.com/opentoonz/opentoonz/issues/5171#issuecomment-1816090526), a `glut32.dll` is required. This library is related to OpenGL.

Does validation VM support it? Shall I add it to package dependency?

**Comment by Dragon1573** [comment]
1. It seems OpenGL is supported by graphics drivers of different GPU manufacturers (Intel, AMD/ATI, NVIDIA).
2. GLUT (aka. OpenGL Unitility Tools) can be seperately installed, but it's released as ZIP archives.
3. There's a FreeGlut package, but it's only available as `*.tar.gz` source code.

I think we are unable to attach dependencies for `DWANGO.OpenToonz.Nightly`. As OpenToonz is a graphical application, users HAVE TO provide a (virtual or not) monitor and install a graphics driver. So I think it's not an error of the package or manifest. It's just a hardware related problem only happen on Manual Validation VM.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**